### PR TITLE
Update gRPC docs.

### PR DESCRIPTION
**Pull request by anuraaga**
- client
- gRPC-web
- Unframed
- Blocking logic

### CODE REVIEW

File: site/src/sphinx/client-grpc.rst [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+.. _CompletableFuture: https://docs.oracle.com/javase/8/docs/api/index.html?java/util/concurrent/CompletableFuture.html
+.. _Futures: https://google.github.io/guava/releases/21.0/api/docs/com/google/common/util/concurrent/Futures.html
+.. _ListenableFuture: https://google.github.io/guava/releases/21.0/api/docs/com/google/common/util/concurrent/ListenableFuture.html
+.. _gRPC: http://www.grpc.io/
+.. _futures-extra: https://github.com/spotify/futures-extra
+
+.. _client-grpc:
+
+Calling a gRPC service
+========================
```
Comment (trustin) [inline site/src/sphinx/client-grpc.rst]: Mismatching number of `=`s

File: site/src/sphinx/client-grpc.rst [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+Let's assume we have the following gRPC_ service definition:, served at ``http://127.0.0.1:8080/``, just like
```
Comment (trustin) [inline site/src/sphinx/client-grpc.rst]: `definition:,` -> `definition,` (colon)

**Comment by anuraaga** [comment]
Fixed

**Comment by trustin** [comment]
Thanks, @anuraaga !

**Pull request merged**